proto/cometbft/services/block_results/v1beta1/block_results_service.proto

```diff
@@ -1,17 +1,17 @@
 syntax = "proto3";
 package cometbft.services.block_results.v1beta1;
 
-option go_package = "github.com/cometbft/cometbft/proto/cometbft/services/block_results/v1beta1";
-
 import "cometbft/services/block_results/v1beta1/block_results.proto";
 
+option go_package = "github.com/cometbft/cometbft/proto/cometbft/services/block_results/v1beta1";
+
 /*
-  BlockResultService provides the block results of a given or latestheight.
- */
+   BlockResultService provides the block results of a given or latestheight.
+*/
 service BlockResultsService {
   // GetBlockResults returns the BlockResults of the requested height.
   rpc GetBlockResults(GetBlockResultsRequest) returns (GetBlockResultsResponse);
 
   // GetLatestBlockResults returns the BlockResults of the latest committed height.
-  rpc GetLatestBlockResults(GetLatestBlockResultsRequest) returns (GetBlockResultsResponse);
+  rpc GetLatestBlockResults(GetLatestBlockResultsRequest) returns (GetLatestBlockResultsResponse);
```

Comment:
Here, we have to get into redundancy because in principle, types of different RPC methods may evolve independently, even though we are sure that in this case the return messages will always be the same.

But I sense a deeper design issue: `GetLatestBlockResults` is non-idempotent, unlike `GetBlockResults`, so perhaps it was a good idea to define it separately and not as a special case behaviour for the other method.

A somewhat more elegant way to work around this is to define a shared message type and reuse it in both responses as the single field.